src/item.cpp

```diff
@@ -3245,16 +3245,41 @@ void item::on_damage( int, damage_type )
 std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int truncate ) const
 {
     std::stringstream ret;
-
+    int dirt_level =  static_cast<int>( get_var( "dirt", 0 ) / 100 );
+    //  std::string dirt_symbol = to_string( static_cast<int>( get_var( "dirt", 0 ) ) );
```

Comment:
Please don't commit commented out code. Either it's useful code and should be run, so it should not be a comment at all. Or it is useless code that should not be there in the first place.